### ISSUE TITLE
[1.60.x] schema download for config verify is down (404)

### ISSUE THREAD

**Issue by tonistiigi** [golangci/golangci-lint#5003]
### Welcome

- [X] Yes, I'm using a binary release within 2 latest [releases](https://github.com/golangci/golangci-lint/releases). Only such installations are supported.
- [X] Yes, I've searched similar [issues on GitHub](https://github.com/golangci/golangci-lint/issues) and didn't find any.
- [X] Yes, I've read the `typecheck` section of the [FAQ](https://golangci-lint.run/welcome/faq/#why-do-you-have-typecheck-errors).
- [X] Yes, I've tried with the standalone [linter](https://golangci-lint.run/usage/linters/) if available (e.g., gocritic, go vet, etc.).
- [X] I agree to follow this project's [Code of Conduct](https://github.com/golangci/golangci-lint?tab=coc-ov-file)

### Description of the problem

```
» golangci-lint config verify
Failed executing command with error: [.golangci.yml] validate: compile schema: jsonschema https://golangci-lint.run/jsonschema/golangci.v1.60.jsonschema.json compilation failed: https://golangci-lint.run/jsonschema/golangci.v1.60.jsonschema.json returned status code 404
```

Seems to work with 1.61 , looks like it broke for 1.60.x after 1.61 was released

### Version of golangci-lint

<details>

```console
$ golangci-lint --version
golangci-lint has version 1.60.3 built with go1.23.0 from c2e095c on 2024-08-22T21:45:24Z
</details>

### Configuration

<details>

```console
# paste configuration file or CLI flags here
```

</details>

### Go environment

<details>

```console
$ go version && go env
# paste output here
```

</details>

### Verbose output of running

<details>

```console
$ golangci-lint cache clean
$ golangci-lint run -v
# paste output here
```

</details>

### A minimal reproducible example or link to a public repository

<details>

```go
// add your code here
```

</details>

### Validation

- [X] Yes, I've included all information above (version, config, etc.).

### Supporter

- [ ] I am a sponsor/backer through [GitHub](https://github.com/sponsors/golangci/) or [OpenCollective](https://opencollective.com/golangci-lint)

**Comment by ldez** [comment]
I fixed the problem.

**Comment by tonistiigi** [comment]
@ldez 🤔 

```
» golangci-lint config verify
Failed executing command with error: [.golangci.yml] validate: compile schema: jsonschema https://golangci-lint.run/jsonschema/golangci.v1.60.jsonschema.json compilation failed: https://golangci-lint.run/jsonschema/golangci.v1.60.jsonschema.json returned status code 404
```

**Comment by ldez** [comment]
can you try again?

**Comment by tonistiigi** [comment]
@ldez works now
